03-vectors/vectors-practice-1.Rmd

```diff
@@ -91,6 +110,12 @@ the points that are part of Hobbes on the right side have been mistakenly moved
 up by 25 units. Please fix this problem, so that drawing `p1x` with `p1y` results 
 in a more plausible rendering of Hobbes. 
 
+```{r}
+new.p1y <- p1y
```

Comment:
This was a good idea to re-assign these vectors to new variables - I did not do this and ended up having to re-run previous statements, so tedious...